vhd/packer/configure-windows-vhd.ps1

```diff
@@ -91,6 +91,7 @@ function Get-FilesToCacheOnVHD
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.11-azs/windowszip/v1.15.11-azs-1int.zip",
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.12-azs/windowszip/v1.15.12-azs-1int.zip",
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.16.11-azs/windowszip/v1.16.11-azs-1int.zip",

```

Comment:
We should get rid of this 1.16.11 artifact as we're disabling 1.16.11 for Windows in this PR

Reply:
change made